app/screens/receive-bitcoin-screen/use-receive-bitcoin.ts

```diff
@@ -455,6 +457,56 @@ export const useReceiveBitcoin = () => {
     readablePaymentRequest = `${pr.info.data.username}@${lnAddressHostname}`
   }
 
+  const receiveViaNFC = async (
+    destination: ReceiveDestination,
+    settlementAmount: MoneyAmount<"BTC">,
+  ) => {
+    const { callback, defaultDescription, k1 } = destination.validDestination
+    const { data } = await lnInvoiceCreate({
+      variables: {
+        input: {
+          walletId: prcd.receivingWalletDescriptor.id,
+          amount: settlementAmount.amount,
+          memo: prcd.memo || defaultDescription,
+        },
+      },
+    })
+
+    if (!data) {
+      throw new Error("No data returned from lnInvoiceCreate")
```

Comment:
is this error handled properly upstream? or could this crash the mobile app?

Reply:
I made this to display alert instead